Documentation/beta-release-notes.md

```diff
@@ -0,0 +1,62 @@
+.NET Command Line (CLI) tools - BETA Readme
+===========================================
+
+# Breaking changes
+There are two main breaking changes in beta. 
+
+1. Versioning change - we have changed the way we version the packages and introduced a new branching mechanism. You can see more in #668. 
+2. Nightly and dev packages cannot co-exist with release packages on one box. 
+
+The solution for both of these is uninstall:
+
+1. Delete the previous versions of the package
+2. Delete the non-release version of the package
+
+# Uninstalling dotnet
+Due to the above breaking change, we reccommend that you uninstall the pre-Beta bits. Below are instructions for all of the supported platforms. 
+
+## Windows
+Simply uninstall the MSI using Control Panel. 
+
+## Ubuntu
+The best way to remove the package is to purge it with the following command: `sudo apt-get purge [package-name]`. This will remove all of the dotnet that was installed via apt-get. The `[[ackage-name]` above will most likely be `dotnet` but if you installed dev or nightly it will be `dotnet-dev` or `dotnet-nightly` respectivelly.  
```

Comment:
What do you think about including just a blanket list people can copy/paste here.

e.g. To get the new bits:
```bash
sudo apt-get purge dotnet dotnet-dev dotnet-nightly
sudo apt-get update
sudo apt-get install dotnet
```